docs/ipdk-dpdk.md

```diff
@@ -0,0 +1,123 @@
+# Building IPDK networking recipe for DPDK target
+
+## Overview
+
+This document describes how to build and run IPDK networking recipe on DPDK target.
+
+## Setup
+
+### Build P4 SDE for DPDK
+```bash
+git clone --recursive git@github.com:p4lang/p4-dpdk-target.git p4sde
+```
+
+For build instructions, refer [P4 SDE Readme](https://github.com/p4lang/p4-dpdk-target/blob/main/README.md#building-and-installing)
+
+### Build and install infrap4d dependencies
+
+```bash
+git clone --recursive git@github.com:ipdk-io/networking-recipe.git ipdk.recipe
+cd ipdk.recipe
+export IPDK_RECIPE=`pwd`
+cd $IPDK_RECIPE/setup
+cmake . -DCMAKE_INSTALL_PREFIX=`path for installing dependencies`
+make
+```
+
+*Note*: If running as non-sudo user, provide `USE_SUDO = ON` option to cmake config and also use `sudo` for ldconfig
+
+### Build Networking Recipe
+
+#### Set environment variables
+ - export DEPS_INSTALL_PATH=`absolute path for installing dependencies`
+ - export SDE_INSTALL=`absolute path for p4 sde install built in previous step`
+ - export LD_LIBRARY_PATH=$IPDK_RECIPE/install/lib/:$SDE_INSTALL/lib:$SDE_INSTALL/lib64:$DEPS_INSTALL_PATH/lib:$DEPS_INSTALL_PATH/lib64:$LD_LIBRARY_PATH
+ 
+
+#### Compile the recipe
+```bash
+cd $IPDK_RECIPE
+./make-all.sh --target=dpdk
+```
```

Comment:
By default, `make-all.sh` will create the `install` directory under the networking recipe. The following examples assume you have done this. If not, you will need to specify the appropriate path instead of `./install`.

Reply:
Good point. Added a note for the same